Easydict/App/Localizable.xcstrings

```diff
@@ -2660,6 +2187,7 @@
       }
     },
     "select_translate" : {
+      "extractionState" : "manual",
```

Comment:
I found some duplicated localized strings, in addition to the new and old menu bar titles, there are also global shortcuts in the settings page, we should remove the duplicated values.

It's also ok to remove them in another PR.

<img width="1411" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/89434aa1-f943-4241-bec0-fd70df346a74">

Reply:
okay